core/templates/pages/exploration-player-page/layout-directives/learner-local-nav.component.html

```diff
@@ -29,6 +29,7 @@
   <li class="nav-item">
     <a *ngIf="username" (click)="showFlagExplorationModal()"
        ngbTooltip="{{ 'I18N_PLAYER_REPORT_TOOLTIP' | translate }}" placement="bottom"
+       (keydown)="reportExploration($event)"
```

Comment:
I think adding a `href="#"` attribute here should be sufficient to fix the issue.